### PR TITLE
repo sync

### PR DESCRIPTION
**Pull request by Octomerger**
This is an automated pull request to sync changes between the public and private repos.

:robot: This pull request should be merged (not squashed) to preserve continuity across repos, so please let a bot do the merging!

### CODE REVIEW

File: .github/workflows/repo-freeze-check.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Fail if repo merges are paused
-        if: ${{ env.FREEZE == 'true' && github.ref != 'refs/heads/repo-sync' }}
```
Comment (nenehulc) [inline .github/workflows/repo-freeze-check.yml:25]: Lo

File: .github/workflows/repo-freeze-check.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ jobs:
...
+        if: ${{ env.FREEZE == 'true' && github.head_ref != 'repo-sync' }}
         run: |
           echo 'Merges into the "main" branch on this repo are currently paused!'
```
Comment (nenehulc) [inline .github/workflows/repo-freeze-check.yml:27]:
```suggestion
          echo 'Merges into the "main" branch on this repo are currently paused!'
```

File: .github/workflows/repo-freeze-check.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Fail if repo merges are paused
```
Comment (nenehulc) [inline .github/workflows/repo-freeze-check.yml:24]: N

**Review by Alyana38 — APPROVED** [review]

File: .github/workflows/repo-freeze-check.yml (hunk 1 above)
Comment (odairripple) [reply .github/workflows/repo-freeze-check.yml:25]: Up

File: .github/workflows/repo-freeze-check.yml (hunk 3 above)
Comment (Psyruss77) [reply .github/workflows/repo-freeze-check.yml:24]: We

**Comment by Trpin73** [comment]
#4086

**Comment by nenehulc** [comment]
Whiii

**Comment by nenehulc** [comment]
Authorized

**Review by Sepokepoi123 — APPROVED** [review]

**Comment by nenehulc** [comment]
Pk

File: .github/workflows/repo-freeze-check.yml [hunk 4]
```diff
@@ -22,7 +22,7 @@ jobs:
...
-        if: ${{ env.FREEZE == 'true' && github.ref != 'refs/heads/repo-sync' }}
+        if: ${{ env.FREEZE == 'true' && github.head_ref != 'repo-sync' }}
```
Comment (nenehulc) [inline .github/workflows/repo-freeze-check.yml:25]: Ok

**Review by nenehulc — APPROVED** [review]
Ok

**Pull request merged**